scripts/build/publish.sh

```diff
@@ -8,8 +8,8 @@ EXTRA_OPTS="$@"
 
 # Right now we hack this in into the publish script.
 # Eventually we might want to keep a list of all previous releases somewhere.
-_releaseNoteUrl="https://community.grafana.com/t/release-notes-v6-0-x/14010"
-_whatsNewUrl="http://docs.grafana.org/guides/whats-new-in-v6-0/"
+_releaseNoteUrl="https://community.grafana.com/t/release-notes-v7-0-x/29381"
```

Comment:
Is this script still in use or should it be removed?

Reply:
@marefr I think it's probably decommissioned, but haven't taken the effort of really checking.